app/views/register/shared/_form-error-summary.haml

```diff
@@ -0,0 +1,7 @@
+%div.error-summary{:role => 'alert', :'aria-labelledby' => 'error-summary-heading', :tabindex => '-1'}
```

Comment:
I think we should move this outside of the "registers" directory. I would suggest the layouts directory